nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -367,6 +367,32 @@ in {
         The nextcloud-occ program preconfigured to target this Nextcloud instance.
       '';
     };
+
+    extraOptions = mkOption {
+      type = with types; attrsOf anything;
+      default = {};
+      description = ''
+        Extra options which should be appended to nextcloud's config.php file.
+      '';
+      example = {
```

Comment:
IIRC using `literalExample` makes better formatting in the manual.

Reply:
It's correct though becaues it gives you better formatting.

Reply:
Please use `literalExample '' /* nix value */ ''` here.

Reply:
done